src/scss/components/events-highlights.scss

```diff
@@ -0,0 +1,83 @@
+.event {
+  $root: &;
+  @extend .container;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 80px;
+
+  &.feature {
+    padding-bottom: 100px;
+    #{$root}__title {
+      font-size: $mobile-title;
+      order: -1;
+    }
+  }
+
+  > * {
```

Comment:
Please don't apply properties to all of the element children, as this could lead to unexpected behaviors. Wrap the elements you want to affect inside a container and using a class, apply the properties. Besides, the only property that is affecting the elements is the `padding`, so you wouldn't need the other ones.

Reply:
resolved